.github/workflows/check-semver.yml

```diff
@@ -19,14 +19,14 @@ jobs:
         with:
           cache-on-failure: true
 
+      - name: install parity-publish
+        run: cargo install parity-publish@0.6.0
+
       - name: Rust compilation prerequisites
         run: |
-          rustup default nightly-2024-03-01
-          rustup target add wasm32-unknown-unknown --toolchain nightly-2024-03-01
-          rustup component add rust-src --toolchain nightly-2024-03-01
-
-      - name: install parity-publish
-        run: cargo install parity-publish@0.5.1
+          rustup default $(parity-publish semver --minimum-nightly-rust-version)
+          rustup target add wasm32-unknown-unknown --toolchain $(parity-publish semver --minimum-nightly-rust-version)
```

Comment:
```suggestion
```
WASM should not be needed since the runtime WASM build is skipped.

Reply:
This was an issue before. It may be because when building docs it uses all targets? Is this a new thing or should i recheck if it works without wasm?

Reply:
Hm then lets keep it in for now.